Remotion/Data/DomainObjects.UnitTests/Factories/BaseConfiguration.cs

```diff
@@ -118,5 +124,29 @@ public FakeDomainObjectsConfiguration GetDomainObjectsConfiguration ()
     {
       return new FakeDomainObjectsConfiguration(_mappingLoaderConfiguration, _storageConfiguration, _queryConfiguration);
     }
+
+    public void DisableConnections ()
+    {
+      foreach (var rdbmsProviderDefinition in _storageConfiguration.StorageProviderDefinitions.OfType<RdbmsProviderDefinition>())
+      {
+        var connectionString = rdbmsProviderDefinition.ConnectionString;
+        if (string.IsNullOrEmpty(connectionString))
+          continue;
+
+        _originalConnectionStrings[rdbmsProviderDefinition] = connectionString;
+        PrivateInvoke.SetNonPublicField(rdbmsProviderDefinition, "_connectionString", string.Empty);
```

Comment:
Missed this earlier: please use a identifiable string instead of empty:
"DATABASE_DISABLED_IN_UNITEST" or something along those lines. Otherwise, the empty string might result in some headscratching down the line.